test/Docfx.Common.Tests/PathUtilityTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using FluentAssertions;
+using Xunit;
+
+namespace Docfx.Common.Tests;
+
+public class PathUtilityTest
+{
+    [Theory]
+    [MemberData(nameof(TestData.AdditionalTests), MemberType = typeof(TestData), DisableDiscoveryEnumeration = false)]
```

Comment:
nit: is there a reason why this marks `DisableDiscoveryEnumeration=false` and the other true? also consider `[InlineData]`?

Reply:
> is there a reason why this marks DisableDiscoveryEnumeration=false

No. It's not necessary settings.
I've created commit that removed `DisableDiscoveryEnumeration=false` settings. (https://github.com/dotnet/docfx/pull/9700/commits/2a32f896a337e33d7caa77511374fc031d14bdf0)

That settings are originally added for test data that can't serialized.
But it's no more needed because test argument is changed to primitive strings.

> would [InlineData] work here?

It can rewrite tests to use `[InlineData]`.
But I', personally prefer to separating test data definitions for following reasons.

- Readability. When using `TheoryData`. It can manually align formatting of test data.
- It can reuse test data set with multiple tests.
- [InlineData] support only primitive types.  
  When using `MemberData`.  It can use dedicated types. And it can change name that displayed on TestExplorer.